src/components/settings/presets/ShipPresets.ts

```diff
@@ -0,0 +1,74 @@
+import { SelectedDistributor, SelectedWeapons, WeaponType } from '../../../typescript/data/dataFormat';
+
+// The below import is unused be necessary due to a Svelte issue with enums, described here:
```

Comment:
Minor type: Should be `is unused but necessary`

Reply:
Fixed